common/beerocks/bcl/source/beerocks_logging.cpp

```diff
@@ -293,7 +293,7 @@ const std::string logging::syslogFormat("<%thread> %fbase[%line] --> %msg");
 logging::logging(const std::string config_path, std::string module_name)
     : m_module_name(module_name), m_logfile_size(LOGGING_DEFAULT_MAX_SIZE),
       m_levels(LOG_LEVELS_GLOBAL_DEFAULT), m_syslog_levels(LOG_LEVELS_SYSLOG_DEFAULT),
-      m_stdout_enabled("true"), m_syslog_enabled("false")
+        m_log_files_enabled(true), m_log_files_auto_roll(true), m_stdout_enabled(true), m_syslog_enabled(false)
```

Comment:
Personally I am not a big fan of initializing class members in the initializer list, and do it only if I have to.
If it's possible, initialize the members on the class definition (.h file).

Reply:
Agree.